src/cleanfigure.m

```diff
@@ -50,15 +56,18 @@ function cleanfigure(varargin)
 
   % Set up command line options.
   m2t.cmdOpts = m2tInputParser;
-  m2t.cmdOpts = m2t.cmdOpts.addParamValue(m2t.cmdOpts, 'minimumPointsDistance', 1.0e-10, @isnumeric);
   m2t.cmdOpts = m2t.cmdOpts.addParamValue(m2t.cmdOpts, 'handle', gcf, @ishandle);
+  m2t.cmdOpts = m2t.cmdOpts.addParamValue(m2t.cmdOpts, 'targetResolution', [15 9 600], @(a)numel(a) == 3);
```

Comment:
Please just write a proper function to validate this.
```matlab
function bool = isValidTargetResolution(val)
    bool = isnumeric(val) && numel(v) == 3;
```
I haven't look at the rest how `NaN` are handled, but if those cause failures, they should be caught at this location as well.